pants

```diff
@@ -137,10 +141,27 @@ function bootstrap_pants {
   echo "${PANTS_BOOTSTRAP}/${target_folder_name}"
 }
 
+function create_pants_ini_if_missing {
+  if [[ -f "pants.ini" ]]; then
+    return 0
+  fi
+  log "Config file \`pants.ini\` not detected in the repository root. Creating with sensible defaults.\n"
+  touch "pants.ini"
+  pants_version="$(./pants -V)"
+  log "* Pinning \`pants_version\` to the most recent stable release: ${pants_version}."
```

Comment:
After 1.15.0 is released, we'll also automatically pin `pants_runtime_python_version`!

```
* Pinning `pants_runtime_python_version` to Python 2.7. Ensure you have this interpreter available everywhere you plan to use Pants!
```

Will probably try to default to Python 3.6, then 3.7, then 2.7, so that dropping Py27 after 1.17.0 is a bit less surprising. TBD..

Reply:
I think that might confuse people here, because there is nothing actionable for them to be able to do.

We're only about 1-3 weeks away from 1.15.0, depending on how many RCs we have, so this will be fixed soon!